### ISSUE TITLE
bug(MatTabsModule):  tab nav bar failing axe accessibility (aria-required-children)

### ISSUE THREAD

**Issue by hankwarner** [angular/components#26733]
### Is this a regression?

- [ ] Yes, this behavior used to work in the previous version

### Description

The example titled "Basic use of the tab nav bar" on `MatTabsModule` ([https://material.angular.io/components/tabs/examples](https://material.angular.io/components/tabs/examples)) fails axe accessibility test. 

Violation is raised by the following:
```
<nav _ngcontent-gae-c190="" mat-tab-nav-bar="" class="mat-mdc-tab-nav-bar mat-mdc-tab-header mat-mdc-tab-nav-bar-stretch-tabs mat-primary" role="tablist">
```

Violation message from axe-cli:
```
  Violation of "aria-required-children" with 2 occurrences!
    Ensures elements with an ARIA role that require child roles contain them. Correct invalid elements at:
     - #component-viewer
     - .mat-mdc-tab-nav-bar-stretch-tabs
    For details, see: https://dequeuniversity.com/rules/axe/4.6/aria-required-children
```

Screenshot:
![Screenshot 2023-03-06 at 5 00 08 PM](https://user-images.githubusercontent.com/40283259/223244669-acd2593b-d00b-46f4-919b-0cd9b8337340.png)

Same violation occurs in Cypress using the axe-cli plugin:
<img width="518" alt="Screenshot 2023-03-06 at 4 57 01 PM" src="https://user-images.githubusercontent.com/40283259/223244118-553eb08e-d50b-4dda-acf4-3c67652724c4.png">

 

### Reproduction

Steps to reproduce:
1. `npm i @axe-core/cli`
2. `axe https://material.angular.io/components/tabs/examples`

Alternatively, using [cypress-axe](https://www.npmjs.com/package/cypress-axe):
```
cy.visit('https://material.angular.io/components/tabs/examples')
cy.injectAxe()
cy.checkA11y()
```

### Expected Behavior

Accessibility rules to pass

### Actual Behavior

Violation of "aria-required-children" occurring on `.mat-mdc-tab-nav-bar-stretch-tabs`

### Environment

- Angular: 15.0.0
- CDK/Material: `"@angular/material": "^15.0.3"`
- Browser(s): Chrome, Chrome headless
- Operating System (e.g. Windows, macOS, Ubuntu): macOS

**Comment by zarend** [comment]
Hello,

Thank you for reporting this issue. I can reproduce by running the axe command. I do not believe this is causing any issues for users, so I'm triaging as P4 priority. This may be a false positive from the checker. Please let us know if you continue to experience problems with this.

-Zach

**Comment by zarend** [comment]
Closing as "not planned".
